ansible_events/drools/vendor/__init__.py

```diff
@@ -0,0 +1,3 @@
+import logging
```

Comment:
This logger doesn't seem to be used. Why is it created here?

Reply:
just a bad copy paste